hex/spec/dependabot/hex/update_checker_spec.rb

```diff
@@ -9,6 +9,10 @@
 require_common_spec "update_checkers/shared_examples_for_update_checkers"
 
 RSpec.describe Dependabot::Hex::UpdateChecker do
+  before do
+    stub_request(:get, hex_url).to_return(status: 200, body: hex_response)
+  end
+
```

Comment:
🔎 For next time: these changes seem to be unrelated to the focus of this PR. It'd be great to keep unrelated changes separate 🙂 That said, the functionality remains, so I think we're okay to go.